src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -825,9 +825,83 @@ public Parent validateParent(Parent parent, Collection<BulkLoadPropertyMappingDT
 							|| foundParent.getCorpName().contains(parent.getLabelPrefix().getLabelPrefix()));
 					if (sameStereoCategory & sameStereoComment
 							& (sameCorpName | (noCorpName & sameCorpPrefixOrNoPrefix))) {
-						// parents match
-						parent = foundParent;
-						break searchResultLoop;
+						// parents match (based on above criteria)
+						boolean equalAliases = true; // assumption is aliases are same unless proven to be false
+
+						// Need to Create String Sets to Do 
+						// "Is alias (str) of parent in aliases of foundParent (set of strs)?" (and vice versa) logic 
+
+						// Note: This is not the most efficient / concise code to do this process
+						// however due to readability and the assumingly small number of aliases 
+						// parents will have this is fine 
+
+						Set<String> parentAliasStrings = new HashSet<>();
+						for(ParentAlias alias : parent.getParentAliases()){
+							if (! alias.isIgnored())
+							{
```

Comment:
nit - Can we stick with { being on the same line as the if or else block since it's that way in most of the code?